nox/virtualenv.py

```diff
@@ -238,7 +268,10 @@ def _resolved_interpreter(self):
             cleaned_interpreter = "python{}".format(xy_version)
 
         # If the cleaned interpreter is on the PATH, go ahead and return it.
-        if py.path.local.sysfind(cleaned_interpreter):
+        if currently_in_virtual_environment and _SYSTEM != "Windows":
```

Comment:
You're ignoring `cleaned_interpreter` here, which seems weird. Why go to the trouble of calculating it and then not use it?

Reply:
Good catch, I didn't intentionally ignore it